hdf5-vol-daos.spec

```diff
@@ -11,7 +11,6 @@
 %global vol_major 1
 %global vol_minor 1
 %global vol_bugrelease 0
-%global vol_prerelease rc4
```

Comment:
Can you leave this in but commented out so that the next time a prerelease is used, it's obvious where it goes?

```
#%%global vol_prerelease rc4		
```

Reply:
added